lib/chartmogul/metrics/activity.rb

```diff
@@ -3,33 +3,37 @@
 module ChartMogul
   module Metrics
     class Activity < ChartMogul::Object
-      readonly_attr :id
       readonly_attr :description
       readonly_attr :type
       readonly_attr :date, type: :time
       readonly_attr :activity_arr
       readonly_attr :activity_mrr
       readonly_attr :activity_mrr_movement
       readonly_attr :currency
-      readonly_attr :currency_sign
       readonly_attr :subscription_external_id
+      readonly_attr :plan_external_id
+      readonly_attr :customer_name
+      readonly_attr :customer_uuid
+      readonly_attr :customer_external_id
+      readonly_attr :billing_connector_uuid
+      readonly_attr :uuid
 
-      def self.all(customer_uuid, options = {})
-        ChartMogul::Metrics::Activities.all(customer_uuid, options)
+      def self.all(options = {})
+        ChartMogul::Metrics::Activities.all(options)
       end
     end
 
     class Activities < APIResource
       set_resource_name 'Activities'
-      set_resource_path '/v1/customers/:customer_uuid/activities'
+      set_resource_path '/v1/activities'
 
       include Concerns::Entries
-      include Concerns::Pageable
```

Comment:
This and removing some fields are backwards-incompatible changes. I recommend adding a short migration guide summarizing the needed changes in code to upgrade to v2.

Reply:
Done. I added a brief description. This file now serves the new activity end point v1/activities. The old file which served results from v1/customers/:uuid/activities has now been moved to lib/chartmogul/metrics/customers/activity.rb

Reply:
@SrikalaB, sorry for not being clear enough, I meant to add it into the project `README` or creating a file in the project, not to comment on it in the PR description where it's fairly hidden.
For an example look here: https://github.com/mperham/sidekiq/blob/master/3.0-Upgrade.md

Reply:
Added the upgrade doc and tests are also passing.